src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2411,7 +2433,6 @@ public override BoundNode VisitIncrementOperator(BoundIncrementOperator node)
                 // PROTOTYPE(NullableReferenceTypes): Update conversion method based on operand type.
                 if (node.OperandConversion.IsUserDefined && (object)node.OperandConversion.Method != null && node.OperandConversion.Method.ParameterCount == 1)
                 {
-                    ReportArgumentWarnings(node.Operand, operandResult.Type, node.OperandConversion.Method.Parameters[0]);
```

Comment:
>ReportArgumentWarnings(node.Operand, operandResult.Type, node.OperandConversion.Method.Parameters[0]); [](start = 20, length = 102)

It is not obvious why this is no longer needed

Reply:
`ApplyConversion` reports argument warnings for user defined conversion operators.

---
In reply to: [183865472](https://github.com/dotnet/roslyn/pull/26356#discussion_r183865472) [](ancestors = 183865472)